lib/active_merchant/billing/gateways/conekta.rb

```diff
@@ -173,22 +173,27 @@ def parse(body)
         JSON.parse(body)
       end
 
-      def headers(meta)
+      def headers(options)
         {
-          "Accept" => "application/vnd.conekta-v#{options[:version]}+json",
+          "Accept" => "application/vnd.conekta-v#{@options[:version]}+json",
```

Comment:
This has to become `@options` to distinguish itself from the local `options` variable that is shadowing the `options` method, right? If so, it's correct; just confused by the diff.

Reply:
Yep, since I need the local options also, I had to make this distinction.